tests/test_grain.py

```diff
@@ -217,23 +217,6 @@ def test_empty_grain_with_meta(self):
         ])
         self.assertEqual(repr(grain), "Grain({!r})".format(meta))
 
-    def test_empty_grain_negative_time_exception(self):
```

Comment:
@alexrawcliffe the code will no longer throw a ValueError if given  a negative TimeOffset, hence this test became redundant.